spring/spring5/spring5-remote/src/test/java/org/infinispan/spring/remote/session/RemoteApplicationPublishedBridgeTest.java

```diff
@@ -36,7 +37,9 @@ public void beforeClass() {
       embeddedCacheManager = TestCacheManagerFactory.createCacheManager();
       hotrodServer = HotRodTestingUtil.startHotRodServer(embeddedCacheManager, 19723);
       ConfigurationBuilder builder = new ConfigurationBuilder();
-      builder.addServer().host("localhost").port(hotrodServer.getPort());
+      builder.addServer().host("localhost").port(hotrodServer.getPort())
+            .marshaller(new JavaSerializationMarshaller())
```

Comment:
we need to specify this all the time to make it work now ?

Reply:
Atm yes, as the new default client marshaller (ProtoStreamMarshaller) does not know how to marshall the spring classes, hence we have to use Serialization.

I tried to make it so that the `JavaSerializationMarshaller` was configured by default just for spring/remote, however I ran into issues with some of the tests as it did not like that the configuration differed from the provided properties. Any thoughts on where is the appropriate place to automatically configure the marshaller for spring/remote?

Reply:
Let me grab a closer look for this. We can't let the user handle this stuff, it's internal IMO

Reply:
I need to test this with Infinispan SB starter